2019-12-28 01:42:18.3532 INFO Creating connection to RabbitMQ
2019-12-28 01:42:24.5675 INFO [REQUEST INFO] Method: GET Scheme: https CotentType:  Target: localhost:5001/task/home/throwEx
2019-12-28 01:42:24.6883 ERROR Something went wrong. Publishing exception to RabbitMq:
Exception source: Task.Api
ProblemCodeChallenge.Common.Exceptions.CustomException: Manual testing publish msg to RabbitMq
   at Task.Api.Controllers.HomeController.ThrowEx() in /home/covilo/workspace/projects/diplomski/CodeChallenge/src/CodeChallenge.Services.Task/Task.Api/Controllers/HomeController.cs:line 15
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at CodeChallenge.Common.RequestInvoker.RequestInvokerMiddleware.InvokeAsync(HttpContext httpContext) in /home/covilo/workspace/projects/diplomski/CodeChallenge/src/CodeChallenge.Common/RequestInvoker/RequestInvokerMiddleware.cs:line 28
2019-12-28 01:42:24.9382 INFO Publishing message to RabbitMQ
2019-12-28 01:46:27.0374 INFO Creating connection to RabbitMQ
2019-12-28 01:46:38.8571 INFO [REQUEST INFO] Method: GET Scheme: https CotentType:  Target: localhost:5001/task/home/throwEx
2019-12-28 01:46:39.0051 ERROR Something went wrong. Publishing exception to RabbitMq:
Exception source: Task.Api
ProblemCodeChallenge.Common.Exceptions.CustomException: Manual testing publish msg to RabbitMq
   at Task.Api.Controllers.HomeController.ThrowEx() in /home/covilo/workspace/projects/diplomski/CodeChallenge/src/CodeChallenge.Services.Task/Task.Api/Controllers/HomeController.cs:line 15
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at CodeChallenge.Common.RequestInvoker.RequestInvokerMiddleware.InvokeAsync(HttpContext httpContext) in /home/covilo/workspace/projects/diplomski/CodeChallenge/src/CodeChallenge.Common/RequestInvoker/RequestInvokerMiddleware.cs:line 28
2019-12-28 01:46:39.2461 INFO Publishing message to RabbitMQ
2019-12-28 02:02:40.8789 INFO Creating connection to RabbitMQ
2019-12-28 02:02:46.6692 INFO [REQUEST INFO] Method: GET Scheme: https CotentType:  Target: localhost:5001/task/home/throwEx
2019-12-28 02:02:46.8367 ERROR Something went wrong. Publishing exception to RabbitMq:
Exception source: Task.Api
ProblemCodeChallenge.Common.Exceptions.CustomException: Manual testing publish msg to RabbitMq
   at Task.Api.Controllers.HomeController.ThrowEx() in /home/covilo/workspace/projects/diplomski/CodeChallenge/src/CodeChallenge.Services.Task/Task.Api/Controllers/HomeController.cs:line 15
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at CodeChallenge.Common.RequestInvoker.RequestInvokerMiddleware.InvokeAsync(HttpContext httpContext) in /home/covilo/workspace/projects/diplomski/CodeChallenge/src/CodeChallenge.Common/RequestInvoker/RequestInvokerMiddleware.cs:line 28
2019-12-28 02:02:47.0100 INFO Publishing message to RabbitMQ
